spec/rails/config/initializers/devise.rb

```diff
@@ -0,0 +1,28 @@
+Devise.setup do |config|
+  config.secret_key = '57b5d0a02bc1e025c5077d9a62d329f66432ba975d3'
```

Comment:
![](https://media1.tenor.com/images/56d3394999ae035ad88eeb9f360476e8/tenor.gif?itemid=13620240)
Until I read this was in spec 😆